.changeset/bright-worms-grin.md

```diff
@@ -7,4 +7,6 @@ Adds following chains;
 * Core testnet
 * Meter 
 * Meter testnet
-* Rari Arbitrum Sepolia testnet
\ No newline at end of file
+* RARI Arbitrum Sepolia testnet
+
+Fixed RARI Chain name
```

Comment:
Looks a bit weird. Can you check this preview https://github.com/api3dao/chains/blob/20a4f3962b4f9972483e25250c2578a060294d10/.changeset/bright-worms-grin.md?